tests/test_build/math_header_item.html

```diff
@@ -1,4 +1,4 @@
-<li class="nav-item current active">
+<li class="nav-item pst-header-nav-item current active">
```

Comment:
I didn't add this class in this PR, but this PR branches off of the feature-focus branch, which added this class but didn't update the tests yet.